### DISCUSSION TITLE
How to move tab bar to the top

### DISCUSSION THREAD

**Discussion by tembbo** [zellij-org/zellij#3613] [Q&A] [answered]
![OnPaste 20240919-170427](https://github.com/user-attachments/assets/24595aa3-4060-44ac-9f0b-bef1272432ab)

How do you move the status bar (in red) to the top.

**Comment by Lockszmith-GH** [comment, marked as answer]
You can change the default layout you are using.

Further reading:
* #3077
* [about editing layouts #2445](https://github.com/zellij-org/zellij/discussions/2445#discussioncomment-5888438)

And documentation:
* https://zellij.dev/documentation/overview.html?default-configuration
* https://zellij.dev/documentation/layouts
* https://zellij.dev/documentation/plugin-aliases
